src/NHibernate/Async/Engine/CollectionEntry.cs

```diff
@@ -70,18 +70,5 @@ public async Task PreFlushAsync(IPersistentCollection collection, CancellationTo
 			reached = false;
 			processed = false;
 		}
-
-		public Task<ICollection> GetOrphansAsync(string entityName, IPersistentCollection collection, CancellationToken cancellationToken)
```

Comment:
This is a breaking change.  Need to bring it back.

Reply:
Done. I've noticed one more lost protected `GetOrphansAsync` method. I've also realized that after my latest changes `IdentityRemove` method is no more used anywhere. So I've removed all the changes in it and made it Obsolete